CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * Enhancement: Resolve in-app-includes and in-app-excludes parameters from the external configuration
 * Enhancement: Make InAppIncludesResolver public (#1084)
 * Ref: Change "op" to "operation" in @SentrySpan and @SentryTransaction
+* Enhancement: Add the ability to register multiple OptionsConfiguration beans (#1093)
```

Comment:
L: can we just call it `feat:` like in the other repos at Sentry?
`Enhancement` takes up a lot of space of the changelog

Reply:
Feature is something different than enhancement AFAIK. Perhaps "enc"?